modules/luci-compat/luasrc/view/cbi/wireless_modefreq.htm

```diff
@@ -129,7 +153,11 @@
 
 		cbi_set_values(mode, modes);
 
-		if (/VHT20|VHT40|VHT80|VHT160/.test(<%= luci.http.write_json(self.map:get(section, "htmode")) %>))
+		if (/EHT20|EHT40|EHT80|EHT160|EHT320/.test(<%= luci.http.write_json(self.map:get(section, "htmode")) %>))
```

Comment:
Also, should EHT240 be in this PR also? (160+80)

Is this something that is reported by your hardware as supported?

Reply:
I haven't seen an openwrt wifi7 device supported with EHT240 .. so I would suggest we park this one until there is one around that can be tested

Reply:
Hmm. Understood. The spec defines EHT240 as valid, so adding it here doesn't hurt. Even if the iwinfo and related bits don't recognise it yet. Up to you.

Reply:
note also I don't think the rpcd / iwinfo changes considered EHT240

Reply:
so ... I am not particularly bothered after all is just another element in the list .... but concerned of potential side effects